src/jni/jvm.h

```diff
@@ -10,24 +10,32 @@ namespace jni {
 
     class Jvm {
     public:
+        enum Type {
+            HOTSPOT,
+            GRAAL,
```

Comment:
I think it's confusing to use Graal for native images as you could also use Graal as a regular JVM, which in this case would fit in the Hotspot value.
Maybe  JVM, NATIVE, and ART ?

Reply:
Instead of NATIVE, maybe something more specific - `NATIVE_IMAGE`.

Reply:
`JVM`, `NATIVE_IMAGE` and `ART`, is everyone ok with this ?

Reply:
I would even opt for `GRAAL_NATIVE_IMAGE` to be very specific. IMO `NATIVE_IMAGE` is a bit generic